packages/babel-traverse/src/path/ancestry.js

```diff
@@ -175,6 +175,20 @@ export function getAncestry() {
   return paths;
 }
 
+/**
+ * A helper to find if `this` path is an ancestor of @param maybeDescendant
+ */
+export function isAncestor(maybeDescendant) {
+  return !!maybeDescendant.findParent((parent) => parent === this);
```

Comment:
Minor, but these seem backwards. If you implement `#isAncestor` through `isDescendent`, you can work on `this`:

```js
function isAncestor(maybeDescendant) {
  return maybeDescendant.isDescendant(this);
}

function isDescendant(maybeAncestor) {
  return !!this.findParent((parent) => parent === maybeAncestor);
}